Sources/NostrSDK/Events/NostrEvent.swift

```diff
@@ -51,7 +51,7 @@ public class NostrEvent: Codable, Equatable, Hashable {
         case content
         case signature = "sig"
     }
-    
+
     init(id: String, pubkey: String, createdAt: Int64, kind: EventKind, tags: [Tag], content: String, signature: String?) {
```

Comment:
I sort of want to eventually get rid of this initializer. We shouldn't make it easy to handcraft your own event by also allowing you to set `id` and `signature` so freely. They should be computed.